playbook/app/pb_kits/playbook/pb_radio/radio.html.erb

```diff
@@ -1,18 +1,42 @@
-<%= content_tag(:label,
-    aria: object.aria,
-    checked: object.checked,
-    class: object.classname,
-    data: object.data,
-    id: object.id,
-    value: object.value,
-    **combined_html_options) do %>
-
-    <% if content.present? %>
-      <%= content %>
-    <% else %>
-      <%= radio_button_tag object.name, object.value, object.selected, object.input_options %>
+<% if object.custom_children %>
+  <div data-pb-radio-children="true">
```

Comment:
Everything works correctly but a few things: if we use custom children, the top level div now is this, which means the wrapping div is no longer 'pb_radio_kit' . Top level div in our kits should have that classname, we do do this in one other place i believe, the CircleChart, where we add a wrapper. If you want to NOT put pb_radio_kit on the top level div, it should at least have a classname to the effect of pb_radio_kit_wrapper, does that make sense?

I'm assuming moving the pb_radio_kit classname up was effecting the way scss was applying here? Was that why we put the pb_radio_kit class inside the flex?

Reply:
@nidaqg I didn't want to change much because I only had to add a div because the Flex kit does not accept data attributes. If we fix this in the future, we won't need this div anymore.

I think passing an empty class is weird since it wouldn't apply any CSS.

I suggest passing the data attribute to the label content tag. This way, we could remove the div and not use an empty class, which would solve your second comment as well (passing global props to the higher wrapper).

What do you think?

Reply:
I'm good with trying that @carloslimasd ! We want to make sure global props are applying to the right div basically